docs/docs/configuration/alpha_config.md

```diff
@@ -148,7 +148,7 @@ You must remove these options before starting OAuth2 Proxy with `--alpha-config`
 
 | Field | Type | Description |
 | ----- | ---- | ----------- |
-| `skipScope` | _bool_ | Skip adding the scope parameter in login request<br/>Default value is 'false' |
+| `SkipScope` | _bool_ | Skip adding the scope parameter in login request<br/>Default value is 'false' |
```

Comment:
This is a shame, the generator previously was using the serialized version of the name

Reply:
Fixed with https://github.com/oauth2-proxy/tools/pull/4